src/components/Navbar.js

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import '../styles/Navbar.css';
 
 const Navbar = () => (
-  <div className="nav-container">
-    <h1 className="header-title">Bookstore CMS</h1>
-    <nav>
-      <ul>
-        <li><NavLink className="books" to="/">BOOKS</NavLink></li>
-        <li><NavLink className="categories" to="/categories">CATEGORIES</NavLink></li>
-      </ul>
-    </nav>
+  <div className="nav-master">
+    <div className="nav-container">
+      <nav className="nav-group">
+        <h1 className="header-title">Bookstore CMS</h1>
+        <ul>
+          <li><NavLink className="books" to="/">BOOKS</NavLink></li>
+          <li><NavLink className="categories" to="/categories">CATEGORIES</NavLink></li>
+        </ul>
+      </nav>
+      <div className="user" />
```

Comment:
- [ ] [Optional] Well! Your icon is a nice addition, but there are a few adjustments needed to match the design more closely. Currently, you've added a shadow to the icon, which is not present in the design. I recommend removing the shadow. Moreover, kindly ensure that the icon's background color matches the design specifications.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/anyars-encarta/book-store/assets/55840999/c0cbd386-7266-4006-bef7-73a26f662066)| ![image](https://github.com/anyars-encarta/book-store/assets/55840999/4de3e11f-9428-4fa8-9cc4-d0bfff29472f) |

By making these changes, your icon will align better with the intended design, creating a more cohesive and visually appealing user interface. Keep up the good work! 👍🎨✨